### PR TITLE
Fix rendering of emojis as first character in nickname

### LINKED ISSUE #160: Fix rendering of emojis as first character in nickname
Some nickname's first character do not render correctly because they may be of larger size than rest of the characters.

Example:
image

### PR DESCRIPTION
**Pull request by Dalfonzo**
Fixes #160 

image

### CODE REVIEW

**Comment by blenderskool** [comment]
Thanks for contributing a fix @Dalfonzo, the change seems to work perfectly.
Before I can merge this, can you make the following changes?
- Extract this logic as a standalone function `/client/src/utils/`.
- Also add the same fix at this location where the first character from the name is displayed https://github.com/blenderskool/blaze/blob/master/client/src/routes/App/Rooms/components/Room/Room.js#L85

**Comment by Dalfonzo** [comment]
@blenderskool Done! Please do let me know if there's something else I need to add/change.

File: client/src/utils/getFirstCharacterFromString.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns the first character from a string
```
Comment (blenderskool) [inline client/src/utils/getFirstCharacterFromString.js:2]: Minor nitpick:
```suggestion
 * Returns the first character from a string preserving emoji characters
```

File: client/src/utils/getFirstCharacterFromString.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ * @param {String} str Text from which the first character will be extracted
+ * @returns {String} First character from the string provided
+ */
+
```
Comment (blenderskool) [inline client/src/utils/getFirstCharacterFromString.js:6]: Remove this extra line? I'm not sure if JSDoc would be able to identify the function this comment is linked to if there's a line break in between.
```suggestion
```

**Review by blenderskool — APPROVED** [review]
Looks great @Dalfonzo, thanks for the changes.
I've minor nitpicks which would be great if you can resolve 🙂

**Comment by Dalfonzo** [comment]
@blenderskool Done! 🙂

**Comment by blenderskool** [comment]
Thanks @Dalfonzo, will push this change in next version update 🙂

**Pull request merged**